### PR TITLE
[5.0] Add "requires version/extension" comments

### PR DESCRIPTION
**Pull request by NogginBops**
### Purpose of this PR

This adds version information (when a function is introduced or in what extension it's introduced) to all native functions.

This PR has now also been extended to:
* Include the gl.xml file in the repo (similar to the spec files).
* Fix one of the overloaders that broke with the updated spec.
* Update the documentation xml files.

### Testing status

The generated files are in the PR and they seem correct.

The files also compiles.

### CODE REVIEW

File: src/OpenTK.Graphics/OpenGL/Enums.cs [hunk 1]
```diff
@@ -1737,7 +1737,6 @@ public enum All : uint
         MaxVertexAttribRelativeOffset = 33497,
         MaxVertexAttribBindings = 33498,
         VertexBindingBuffer = 36687,
-        DisplayList = 33511,
```
Comment (frederikja163) [inline src/OpenTK.Graphics/OpenGL/Enums.cs:1740]: Why are all these enums getting removed?

File: src/OpenTK.Graphics/OpenGLES3/GL.Overloads.cs [hunk 2]
```diff
@@ -9394,132 +9394,6 @@ public static unsafe void GetProgramResourcefvNV(ProgramHandle program, ProgramI
                     GetProgramResourcefvNV(program, programInterface, index, propCount, props_ptr, count, length_ptr, parameters_ptr);
                 }
             }
-            public static unsafe void PathColorGenNV(PathColor color, PathGenMode genMode, PathColorFormat colorFormat, ReadOnlySpan<float> coeffs)
```
Comment (frederikja163) [inline src/OpenTK.Graphics/OpenGLES3/GL.Overloads.cs:9397]: Any idea why this method is gone now?

**Review by frederikja163 — APPROVED** [review]
There are a few oddities in the output with stuff being removed, but other than that this looks good.

**Review by frederikja163 — APPROVED** [review]
Looks good

**Comment by NogginBops** [comment]
Regarding the removed enums and functions. They where actually removed in #1317 but as that PR never committed the generated files, we are seeing those changes here.
With this I'm going to merge this now.

**Pull request merged**